src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -115,4 +115,12 @@ extern {
     pub fn mprotect(addr: *mut ::c_void, len: ::size_t, prot: ::c_int)
                     -> ::c_int;
     pub fn clock_gettime(clk_id: ::c_ulong, tp: *mut ::timespec) -> ::c_int;
+
+    pub fn pthread_attr_get_np(tid: ::pthread_t,
```

Comment:
do these functions have the same signature on both dragonflybsd and freebsd?  if they do, then please put them in the src/unix/bsd/freebsdlike/mod.rs file instead.  thanks!